### ISSUE TITLE
Support for different model (27C6:0111) /  Dell PN771M

### ISSUE THREAD

**Issue by kitsugo** [helloparthshah/dellstylus-linux-driver#1]
Hello!
Thank you very much for this python driver. Unfortunately I'm having issues getting it to work with a slightly different Dell Stylus.

The device is listed as `'CUST0000:00 27C6:01111 Stylus'` in `libinput list-devices`.
I already changed the code so the driver recognizes the different stylus.
`if evdev.InputDevice(dev).name == 'CUST0000:00 27C6:0111 Stylus':`

The driver seems to correctly deactivate the original stylus-device since the stylus no longer sends out any direct inputs while the script is running.
However, the virtual device that is created likewise also does nothing. Essentially the stylus is just deactivated and the virtual device created by the script seemingly doesn't send any input.

Are there more steps that must be taken besides just running the script? Or is this an issue with the script not being compatible with this model? I'd be very grateful for any support on this issue.

**Comment by helloparthshah** [comment]
You could try running evtest and checking what output signals your stylus is giving out. For instance, your stylus might not be using ABS_MT_POSITION_X for the x position. Once you get the different codes, you can change them in the script to simulate your stylus as well.

**Comment by kitsugo** [comment]
Thank you, I managed to rewrite the driver to mostly support my stylus.

Instead of `ABS_MT_POSITION_X/Y` it uses just `ABS_X/Y`.
I also had to adjust the minimum and maximum of these values to fit my device, since otherwise the calibration would be off.

I even managed to make the stylus button work, however, I was not able to properly implement the button at the other end of the stylus and the stylus rubber feature. The rear-button does not seem to send any input to `evtest` whatsoever, so I suppose it is unusable. Apparently it also only has a function under Windows with certain Windows applications.
The rubber feature would still require some more testing, but the functionality is there. However, after using it once, the default pen input seems to be deactivated and the driver needs to be restarted. I have not found out how to fix this issue yet.

Overall the pen is working fine for most activities with this driver.
If desired, I can send the modified code for the `CUST0000:00 27C6:0111 Stylus`.
